### PR TITLE
fix: changed limits for kafka instance

### PR DESCRIPTION
**Pull request by wtrocki**
Does change limits for kafka instances.

### CODE REVIEW

File: src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx [hunk 1]
```diff
@@ -119,6 +119,15 @@ export const CreateKafkaInstance: FunctionComponent<
     [create]
   );
 
+  if (!isTrial) {
+    ingresEgress = 100;
```
Comment (wtrocki) [inline src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx:123]: For ingressEgress there are two values in doc 100 and 50.

Comment (wtrocki) [reply src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx:123]: This is for standard instance

Comment (jgiardino) [reply src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx:123]: You're correct. The updates we made for [the new UI](https://redhat-developer.github.io/app-services-ui-components/?path=/story/kafka-create-kafka-instance-with-sizes-components-instance-info--quota-instance) split these out into separate values. I'm not sure what our options are for splitting them out for this PR. Maybe this is something that @riccardo-forina can provide a suggestion for.

**Review by ajaypratap003 — APPROVED** [review]
LGTM

**Comment by wtrocki** [comment]
@riccardo-forina will take this over to improve component messaging.

**Pull request merged**